pages/404.php

```diff
@@ -3,7 +3,301 @@
 <head>
   <meta charset="UTF-8">
   <title>404 Page Not Found</title>
-  <link rel="stylesheet" href="./css/404.css">
+  <style>
+  @import url(https://fonts.googleapis.com/css?family=Ubuntu);
```

Comment:
@Sukriti-sood 
It would be better to put the CSS in `404.css` (as it was before) since it increases the size of 404.php.

Reply:
@siv2r  Actually 404.php is used  when wrrong route is accessed and 404.php included link of css and for most of the routes 404.css is not at that place relative to that route (hence css is not loaded for most of the time), At the time , I didn't find  any other option so I included css in style , if you  have other option , then please suggest me.

Reply:
Ohh.. I didn't notice this. Can you try linking the CSS with the absolute path then? like `/kidney-exchange/css/404.css`
let me know if this works

Reply:
I have tried this ,it didn't work.

Reply:
![Screenshot from 2021-06-12 12-23-45](https://user-images.githubusercontent.com/55010599/121767947-20667900-cb79-11eb-819d-b070f03cd7a8.png)

Reply:
you missed the `/` before `kidney-exchange`. This will work I guess since, some of the code uses this method.

For Example,
https://github.com/siv2r/kidney-exchange/blob/35d7fa59216442f8319064eaeef29bf74ef8fef3/pages/displayGlobalMatch.php#L20

Reply:
Ya , its working , my bad